packages/react/src/components/Tabs/Tabs.mdx

```diff
@@ -201,6 +207,51 @@ loaded when the Tab is clicked. In v11, to do this, you can this by setting
 </Tabs>
 ```
 
+### Tabs and the Grid - fullWidth prop
+
+By default, a `Tab` component is only as wide as it's content. This posses difficulties when trying to align tabs to the grid.
+Alternatively, you may choose to use the `fullWidth` prop to allow Tab elements to grow as big wide their container allows.
+Note that this feature is *only available* for `contained` tabs with less than 9 tabs, otherwise the prop will be ignored.
+`fullWidth` paired up with a wrapping `Grid` component will allow for "grid-aware" tabs:
```

Comment:
I think we could add some more docs here no how to get the tabs to align on the grid exactly using this. 

> In order to have the tabs align on grid columns you will want to be sure to make the number of columns match the number of tabs. For example if you have 5 tabs you would want to use. `<Column lg={5} md={5}>`